js/render/nodes/cube-sea-occlusion.js

```diff
@@ -204,7 +209,10 @@ class CubeSeaMaterial extends Material {
       }
 
     vec4 fragment_main() {
-      vec2 depthUv = vec2(gl_FragCoord.x/1680.0, gl_FragCoord.y/1760.0);
+      vec4 depthPosition = (VIEW_ID == 0u) ? LEFT_DEPTH_PROJECTION_MATRIX * LEFT_DEPTH_VIEW_MATRIX * vWorldPosition :
+                          RIGHT_DEPTH_PROJECTION_MATRIX * RIGHT_DEPTH_VIEW_MATRIX * vWorldPosition;
+      vec2 depthPositionHC = depthPosition.xy / depthPosition.w;
```

Comment:
Being a bit graphics-unfamiliar, I'm not sure what this is doing? (e.g. what HC stands for)

Reply:
It stands for "Homogeneous Coordinates"
I just copied it from [here](https://github.com/meta-quest/Meta-OpenXR-SDK/blob/main/Samples/XrSamples/XrPassthroughOcclusion/Src/XrPassthroughOcclusionGl.cpp#L632) :-)